web/packages/teleterm/src/ui/ModalsHost/modals/AuthenticateWebDevice/AuthenticateWebDevice.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Alert from 'design/Alert';
+import { ButtonPrimary, ButtonSecondary } from 'design';
+import Dialog, { DialogContent } from 'design/Dialog';
+import Flex from 'design/Flex';
+import { useAsync } from 'shared/hooks/useAsync';
+
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import { RootClusterAppUri, routing } from 'teleterm/ui/uri';
+
+type Props = {
+  rootClusterUri: RootClusterAppUri;
+  onClose(): void;
+  onAuthorize(): void;
+};
+
+export const AuthenticateWebDevice = ({
+  onAuthorize,
+  onClose,
+  rootClusterUri,
+}: Props) => {
+  const [attempt, run] = useAsync(async () => {
+    await onAuthorize();
+    onClose();
+  });
+  const { clustersService } = useAppContext();
+  const clusterName =
+    clustersService.findCluster(rootClusterUri)?.name ||
+    routing.parseClusterName(rootClusterUri);
+
+  return (
+    <Dialog open={true}>
+      <DialogContent maxWidth="360px">
```

Comment:
Spoke with Kenny, we decided to add the cluster name into the message and specifically chose 360px to try and get the cluster name as the "first" item on the next line. 
![Screenshot 2024-04-23 at 12 36 05 PM](https://github.com/gravitational/teleport/assets/5201977/34afa048-45ea-4d90-98f0-0737a5d785f6)

Reply:
Do we want to make the cluster name bold? We usually do this in other dialogs:
<img width="626" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/034ca2f2-06d1-4b83-be82-91b0f787d953">

Reply:
> specifically chose 360px to try and get the cluster name as the "first" item on the next line

Could you leave a comment that mentions this?